applications/GeoMechanicsApplication/custom_constitutive/coulomb_yield_surface.cpp

```diff
@@ -61,6 +61,36 @@ Vector CoulombYieldSurface::DerivativeOfFlowFunction(const Vector&, CoulombAvera
     return result;
 }
 
+void CoulombYieldSurface::UpdateSurfaceProperties(double InitialFrictionAngle,
+                                                  double FrictionAngleStrengthFactor,
+                                                  double InitialCohesion,
+                                                  double CohesionStrengthFactor,
+                                                  double InitialDilatancyAngle,
+                                                  double DilatancyAngleStrengthFactor,
+                                                  double kappa)
+{
+    mFrictionAngle = this->UpdateFrictionAngle(InitialFrictionAngle, FrictionAngleStrengthFactor, kappa) *
+                     3.14159265358979323846 / 180;
```

Comment:
If you include the <numbers> header, you can find pi there (std::numbers::pi). However, we also have a utility function in core that does the conversion:
```suggestion
    mFrictionAngle = MathUtils<>::DegreesToRadians(this->UpdateFrictionAngle(InitialFrictionAngle, FrictionAngleStrengthFactor, kappa));
```